io.github.dfandrich.gpscorrelate.json

```diff
@@ -26,44 +39,45 @@
           "cleanup": [
               "/bin",
               "/include",
-              "/lib/cmake",
               "/lib/pkgconfig",
               "/lib/*.a",
-              "/share/man"
+              "/share/cmake/exiv2/*",
+              "/share/man/man1/*"
           ],
           "sources": [
               {
                   "type": "archive",
-                  "url": "https://github.com/Exiv2/exiv2/releases/download/v0.27.6/exiv2-0.27.6-Source.tar.gz",
-                  "sha256": "4c192483a1125dc59a3d70b30d30d32edace9e14adf52802d2f853abf72db8a6"
+                  "url": "https://github.com/Exiv2/exiv2/archive/refs/tags/v0.28.2.tar.gz",
+                  "sha256": "543bead934135f20f438e0b6d8858c55c5fcb7ff80f5d1d55489965f1aad58b9"
               }
           ]
         },
         {
             "name": "gpscorrelate-gui",
             "buildsystem": "simple",
             "build-commands": [
-                /* gettext catalogues are dropped if stored in the standard location, so move them */
-                "make -j${FLATPAK_BUILDER_N_JOBS} prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale CFLAGS=\"${CFLAGS} -DENABLE_NLS=1\"",
+                "make -j${FLATPAK_BUILDER_N_JOBS} prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale CFLAGS=\"${CFLAGS} -DENABLE_NLS=1\" CXXFLAGS=\"${CXXFLAGS} -DENABLE_NLS=1\"",
                 "make prefix=${FLATPAK_DEST} localedir=${FLATPAK_DEST}/share/gpscorrelate/locale install install-po",
+                "sed -i 's/Icon=gpscorrelate-gui/Icon=io.github.dfandrich.gpscorrelate/' gpscorrelate.desktop",
```

Comment:
use `destkop-file-edit` instead.

Reply:
This was a hack to figure out why the build is failing. Adding png icons didn't actually seem to make any difference.